llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -5831,6 +5831,16 @@ const TargetRegisterClass *SIInstrInfo::getRegClass(const MCInstrDesc &TID,
                                    IsAllocatable);
 }
 
+const TargetRegisterClass *
+SIInstrInfo::getRegClassForBlockOp(const TargetRegisterInfo *TRI,
+                                   const MachineFunction &MF) const {
+  const MCInstrDesc &ScratchStoreBlockOp =
```

Comment:
Can you expand on why this is retrieved from a specific instruction descriptor?

Reply:
I guess it doesn't have to be. I removed the helper altogether.